scipy/optimize/tests/_cython_examples/meson.build

```diff
@@ -0,0 +1,18 @@
+project('random-build-examples', 'c', 'cpp', 'cython')
+
+py_mod = import('python')
+py3 = py_mod.find_installation(pure: false)
```

Comment:
Minor: this is more idiomatically written as a one-liner since `py_mod` isn't used for anything:
```meson
py3 = import('python').find_installation(pure: false)
```

This comment applies to the other `meson.build` files too.